Makefile

```diff
@@ -43,4 +46,6 @@ upgrade_test:
 
 feature_tests: setup_env remediation_test cleanup_env pivoting_test
 
+feature_tests_upgrade: setup_env_upgrade upgrade_test
```

Comment:
no cleanup ?

Reply:
you could actually define the variables CAPM3RELEASE and CAPIRELEASE here, to avoid modifying all the scripts to set them